app/src/App.js

```diff
@@ -152,13 +203,34 @@ const App = () => {
   // }
   // console.log(set);
 
+  const [graph, setGraph] = React.useState("Birthday Party");
+
+  const onSave = async (e) => {
+    let nodes = tasksx,
+      links = arrows;
+    let saveData = { graph, nodes, links };
+    e.preventDefault(); // prevents refreshing
+    let canAdd = saveData.nodes.length > 0 && graph.length > 0;
+    if (canAdd) {
+      await apiClient.addTask(saveData);
+      // loadTasks();
+      // setTask("");
+    }
+  };
+
   return (
     <>
       <header>
         <h1>Prerequisite Tree</h1>
       </header>
 
-      <main className="App">
+      <main
+        className="App"
+        // style={{
```

Comment:
Why is this commented out? Seems like an appropriate element to apply your background?

Reply:
it’s commented out because it doesn’t work! I pasted the styles into different elements to try to get them to work, then commented out when it did not.